### ISSUE TITLE
Table preset color assistance for official blue

### ISSUE THREAD

**Issue by carrotstien** [ForFunLabs/ElevenPublicBugsAndAsks#271]
dan0956

I’m struggling to get my table this colour blue. Wish it was default and not the red. I know you can change colours but can’t quite get it right..

![image](https://user-images.githubusercontent.com/234703/76126377-ebaa6000-5fcc-11ea-90a6-2a927f6f56df.png)

**Comment by Llamalland** [comment]
Yeah, that is definitely my experience too, this should definitely be a preset.

**Comment by Llamalland** [comment]
Also wouldn't mind this, both for the grey colour and for the lack of the doubles lines: 
![image](https://user-images.githubusercontent.com/38714700/76411929-afb73680-63e2-11ea-9e4e-eea7db72b62b.png)
